lib/clearance/session.rb

```diff
@@ -2,7 +2,7 @@
 
 module Clearance
   class Session
-    REMEMBER_TOKEN_COOKIE = 'remember_token'.freeze
+    REMEMBER_TOKEN_COOKIE = Clearance.configuration.cookie_name.freeze
```

Comment:
I think setting this here as a constant will mean that it won't pick up the change if you change it in something like an initializer, because the Clearance middleware stack will be eagerly loaded. Is that right?